### DISCUSSION TITLE
[IPv6]容器默认没有IPv6网络能力吗？

### DISCUSSION THREAD

**Discussion by youzhazhu** [1Panel-dev/1Panel#4157] [Q&A]
如题，我在1Panel中装了Uptime-Kuma容器。
但是由于有些需要监控的VPS是纯IPv6鸡，然而我发现好像容器没有IPv6网络能力，请问这怎么解决呢？

**Comment by realzolo** [comment]
同样，按照文档创建具有IPV6的docker网络，报错无法创建
<img width="1920" alt="cbd7795e56572c92151bdf055fb2a13" src="https://github.com/1Panel-dev/1Panel/assets/63219216/aee95165-95ba-40bc-8dae-dc7fd12ddaaa">
<img width="1920" alt="21a8188d571282b057dbaaea75da33f" src="https://github.com/1Panel-dev/1Panel/assets/63219216/f49d6019-1c23-4f2d-be84-6f6f844ce5d4">
